plugins/gatsby-source-umich-lib/gatsby-node.js

```diff
@@ -49,7 +49,19 @@ exports.sourceNodes = async (
     baseUrl
   }
 ) => {
-  const { createNode } = actions
+  const { createTypes, createNode } = actions
+  const typeDefs = `
+    type HTML {
+      processed: String
+    }
+
+    type paragraph__hero_panel implements Node {
+      field_caption_text: HTML
+    }
+  `
+
+  createTypes(typeDefs)
```

Comment:
@vagabundeo I don't think it's possible to make fields with depth generically nullable. So added a definition here to make sure `field_caption_text` exists on `paragraph__hero_panel`.

Reply:
eh. this will bite us if we have to do it every time

Reply:
I get what you feel you want, but it's not feasible.

Reply:
I don't see how Gatsby could assume the shape of type definitions is has no information about.